sidecar-shim: README: fix misinterpretation (#11213)
kubevirt/kubevirt: cmd/sidecars/README.md

## Example

Using the current [smbios sidecar](../example-hook-sidecar/) as example. The `smbios.go` is compiled

> and the binary is named `onDefineDomain` and installed in `/usr/bin` in a container that uses
> `sidecar-shim-image` as base which has `/sidecar-shim` as the entrypoint.

Reviewer: ```suggestion
as a binary named `onDefineDomain` and installed under `/usr/bin` in a container that uses
`sidecar-shim-image` as base with an entrypoint of `/sidecar-shim`.
```
